com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -27,7 +27,9 @@ public sealed class ForwardRenderer : ScriptableRenderer
         PostProcessPass m_FinalPostProcessPass;
         FinalBlitPass m_FinalBlitPass;
         CapturePass m_CapturePass;
-
+#if ENABLE_VR && ENABLE_XR_MODULE
+        XROcclusionMeshPass m_XROcclusionMeshPass;
```

Comment:
Is the occlusion mesh pass something we can abstact in ScriptableRenderer base class? This would help not either with Deferred renderer but to also abstract XR specific code for people implementing custom renderers for URP.

Reply:
Not possible to abstract this. Renderer need to be aware of this render pass and configure which depth/stencil buffer this pass will render to.
XRSystem does provide lego pieces of drawing the meshes and occlusion mesh shaders. But it is up to the renderer when to draw and where to draw.